### PR TITLE
International module

### PR DESCRIPTION
**Pull request by rmarinleal**

### CODE REVIEW

File: controllers/front/notify.php [hunk 1]
```diff
@@ -99,6 +97,19 @@ class ClearpayNotifyModuleFrontController extends AbstractController
...
     protected $config;
+    /**
+     * Default API Version per region
+     *
+     * @var array
+     */
+    public $defaultApiVersionPerRegion = array(
+        'AU' => 'v2',
+        'CA' => 'v2',
+        'ES' => 'v1',
```
Comment (tmeliotpg) [inline controllers/front/notify.php:108]: Only `'ES'` ?

File: views/templates/hook/checkout.tpl [hunk 2]
```diff
@@ -146,36 +154,36 @@
...
+                    <div class="clearpay-more-info">
+                        {$MOREINFO_ONE|escape:'htmlall':'UTF-8'}
+                    </div>
+                    <afterpay-price-table
+                            data-type="price-table"
+                            data-amount="{$TOTAL_AMOUNT|escape:'htmlall':'UTF-8'}"
+                            data-price-table-theme="white"
+                            data-locale="{$ISO_COUNTRY_CODE|escape:'htmlall':'UTF-8'}"
+                            data-currency="{$CURRENCY|escape:'htmlall':'UTF-8'}">
+                    </afterpay-price-table>
```
Comment (tmeliotpg) [inline views/templates/hook/checkout.tpl]: The correct implementation is as below see price [table table](https://js.sandbox.afterpay.com/index.html) tab

```html                    
<afterpay-placement
      data-type="price-table"
      data-amount="{$TOTAL_AMOUNT|escape:'htmlall':'UTF-8'}"
      data-price-table-theme="white"
      data-locale="{$ISO_COUNTRY_CODE|escape:'htmlall':'UTF-8'}"
      data-currency="{$CURRENCY|escape:'htmlall':'UTF-8'}">
</afterpay-placement>
```

File: controllers/front/notify.php (hunk 1 above)
Comment (rmarinleal) [reply controllers/front/notify.php:108]: Yes, only one from europe is needed, 'ES' or other from Europe.

File: views/templates/hook/checkout.tpl (hunk 2 above)
Comment (rmarinleal) [reply views/templates/hook/checkout.tpl]: resolved

**Review by pgarcess — APPROVED** [review]

**Pull request merged**